app/Commands/Component/UpdateComponentCommand.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Commands\Component;
+
+use CachetHQ\Cachet\Models\Component;
+
+class UpdateComponentCommand
+{
+    /**
+     * The component.
```

Comment:
Maybe better to say "The component to be updated." like I do in styleci, otherwise it's a bit vague.